src/Flex/Flex.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { FLEX_PROPS } from './Flex.types';
+import { FlexWrapper } from './FlexWrapper.styles';
+
+const Flex = ({ className, children, ...props }) => (
+  <FlexWrapper className={className} {...props}>
+    {children}
+  </FlexWrapper>
+);
+
+Flex.propTypes = {
+  alignItems: PropTypes.oneOf(Object.values(FLEX_PROPS.alignItems)),
+  alignSelf: PropTypes.oneOf(Object.values(FLEX_PROPS.alignSelf)),
+  className: PropTypes.string,
+  /**
+    If `true`, `display: flex;` otherwise `display: block;`
+  */
+  container: PropTypes.bool,
+  direction: PropTypes.oneOf(Object.values(FLEX_PROPS.direction)),
```

Comment:
Looks like this one flew under the radar maybe:

```suggestion
  flexDirection: PropTypes.oneOf(Object.values(FLEX_PROPS.flexDirection)),
```